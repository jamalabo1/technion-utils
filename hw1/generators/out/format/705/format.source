C W 950 1
D X 248 2
A W 994 3
B Z 610 4
C Y 500 5
B X 166 6
C W 509 7
B Z 669 8
A X 882 9
A W 285 10
C Z 697 11
A W 309 12
C W 318 13
C Z 815 14
B X 285 15
B Z 589 16
B W 43 17
C Y 747 18
C Y 346 19
B Y 431 20
B Z 578 21
C X 352 22
A Y 513 23
A Z 164 24
A W 381 25
B W 695 26
D X 360 27
C X 172 28
D Y 768 29
D Y 92 30
C Y 409 31
A X 27 32
D Z 767 33
C W 429 34
D Z 24 35
D Z 337 36
B Z 951 37
C W 906 38
B Y 843 39
B W 443 40
A X 956 41
A W 521 42
C Z 274 43
D Z 566 44
D W 195 45
C X 831 46
B Y 560 47
D Z 266 48
A W 28 49
B Z 116 50
B Z 162 51
D Z 9 52
C Y 96 53
C X 343 54
A Y 185 55
A W 620 56
A Z 65 57
B Z 608 58
B Z 908 59
D W 546 60
A W 462 61
B Y 898 62
C Z 689 63
A X 465 64
A Z 952 65
D Y 443 66
A X 23 67
C X 845 68
B Z 285 69
A W 837 70
C W 917 71
C Z 609 72
C Z 763 73
B Y 116 74
B Z 129 75
D Y 46 76
D X 827 77