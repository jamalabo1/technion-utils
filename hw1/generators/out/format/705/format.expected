Blockchain Info:
1
Sender Name: D
Receiver Name: X
Transaction Value: 827
Transaction timestamp: 77
2
Sender Name: D
Receiver Name: Y
Transaction Value: 46
Transaction timestamp: 76
3
Sender Name: B
Receiver Name: Z
Transaction Value: 129
Transaction timestamp: 75
4
Sender Name: B
Receiver Name: Y
Transaction Value: 116
Transaction timestamp: 74
5
Sender Name: C
Receiver Name: Z
Transaction Value: 763
Transaction timestamp: 73
6
Sender Name: C
Receiver Name: Z
Transaction Value: 609
Transaction timestamp: 72
7
Sender Name: C
Receiver Name: W
Transaction Value: 917
Transaction timestamp: 71
8
Sender Name: A
Receiver Name: W
Transaction Value: 837
Transaction timestamp: 70
9
Sender Name: B
Receiver Name: Z
Transaction Value: 285
Transaction timestamp: 69
10
Sender Name: C
Receiver Name: X
Transaction Value: 845
Transaction timestamp: 68
11
Sender Name: A
Receiver Name: X
Transaction Value: 23
Transaction timestamp: 67
12
Sender Name: D
Receiver Name: Y
Transaction Value: 443
Transaction timestamp: 66
13
Sender Name: A
Receiver Name: Z
Transaction Value: 952
Transaction timestamp: 65
14
Sender Name: A
Receiver Name: X
Transaction Value: 465
Transaction timestamp: 64
15
Sender Name: C
Receiver Name: Z
Transaction Value: 689
Transaction timestamp: 63
16
Sender Name: B
Receiver Name: Y
Transaction Value: 898
Transaction timestamp: 62
17
Sender Name: A
Receiver Name: W
Transaction Value: 462
Transaction timestamp: 61
18
Sender Name: D
Receiver Name: W
Transaction Value: 546
Transaction timestamp: 60
19
Sender Name: B
Receiver Name: Z
Transaction Value: 908
Transaction timestamp: 59
20
Sender Name: B
Receiver Name: Z
Transaction Value: 608
Transaction timestamp: 58
21
Sender Name: A
Receiver Name: Z
Transaction Value: 65
Transaction timestamp: 57
22
Sender Name: A
Receiver Name: W
Transaction Value: 620
Transaction timestamp: 56
23
Sender Name: A
Receiver Name: Y
Transaction Value: 185
Transaction timestamp: 55
24
Sender Name: C
Receiver Name: X
Transaction Value: 343
Transaction timestamp: 54
25
Sender Name: C
Receiver Name: Y
Transaction Value: 96
Transaction timestamp: 53
26
Sender Name: D
Receiver Name: Z
Transaction Value: 9
Transaction timestamp: 52
27
Sender Name: B
Receiver Name: Z
Transaction Value: 162
Transaction timestamp: 51
28
Sender Name: B
Receiver Name: Z
Transaction Value: 116
Transaction timestamp: 50
29
Sender Name: A
Receiver Name: W
Transaction Value: 28
Transaction timestamp: 49
30
Sender Name: D
Receiver Name: Z
Transaction Value: 266
Transaction timestamp: 48
31
Sender Name: B
Receiver Name: Y
Transaction Value: 560
Transaction timestamp: 47
32
Sender Name: C
Receiver Name: X
Transaction Value: 831
Transaction timestamp: 46
33
Sender Name: D
Receiver Name: W
Transaction Value: 195
Transaction timestamp: 45
34
Sender Name: D
Receiver Name: Z
Transaction Value: 566
Transaction timestamp: 44
35
Sender Name: C
Receiver Name: Z
Transaction Value: 274
Transaction timestamp: 43
36
Sender Name: A
Receiver Name: W
Transaction Value: 521
Transaction timestamp: 42
37
Sender Name: A
Receiver Name: X
Transaction Value: 956
Transaction timestamp: 41
38
Sender Name: B
Receiver Name: W
Transaction Value: 443
Transaction timestamp: 40
39
Sender Name: B
Receiver Name: Y
Transaction Value: 843
Transaction timestamp: 39
40
Sender Name: C
Receiver Name: W
Transaction Value: 906
Transaction timestamp: 38
41
Sender Name: B
Receiver Name: Z
Transaction Value: 951
Transaction timestamp: 37
42
Sender Name: D
Receiver Name: Z
Transaction Value: 337
Transaction timestamp: 36
43
Sender Name: D
Receiver Name: Z
Transaction Value: 24
Transaction timestamp: 35
44
Sender Name: C
Receiver Name: W
Transaction Value: 429
Transaction timestamp: 34
45
Sender Name: D
Receiver Name: Z
Transaction Value: 767
Transaction timestamp: 33
46
Sender Name: A
Receiver Name: X
Transaction Value: 27
Transaction timestamp: 32
47
Sender Name: C
Receiver Name: Y
Transaction Value: 409
Transaction timestamp: 31
48
Sender Name: D
Receiver Name: Y
Transaction Value: 92
Transaction timestamp: 30
49
Sender Name: D
Receiver Name: Y
Transaction Value: 768
Transaction timestamp: 29
50
Sender Name: C
Receiver Name: X
Transaction Value: 172
Transaction timestamp: 28
51
Sender Name: D
Receiver Name: X
Transaction Value: 360
Transaction timestamp: 27
52
Sender Name: B
Receiver Name: W
Transaction Value: 695
Transaction timestamp: 26
53
Sender Name: A
Receiver Name: W
Transaction Value: 381
Transaction timestamp: 25
54
Sender Name: A
Receiver Name: Z
Transaction Value: 164
Transaction timestamp: 24
55
Sender Name: A
Receiver Name: Y
Transaction Value: 513
Transaction timestamp: 23
56
Sender Name: C
Receiver Name: X
Transaction Value: 352
Transaction timestamp: 22
57
Sender Name: B
Receiver Name: Z
Transaction Value: 578
Transaction timestamp: 21
58
Sender Name: B
Receiver Name: Y
Transaction Value: 431
Transaction timestamp: 20
59
Sender Name: C
Receiver Name: Y
Transaction Value: 346
Transaction timestamp: 19
60
Sender Name: C
Receiver Name: Y
Transaction Value: 747
Transaction timestamp: 18
61
Sender Name: B
Receiver Name: W
Transaction Value: 43
Transaction timestamp: 17
62
Sender Name: B
Receiver Name: Z
Transaction Value: 589
Transaction timestamp: 16
63
Sender Name: B
Receiver Name: X
Transaction Value: 285
Transaction timestamp: 15
64
Sender Name: C
Receiver Name: Z
Transaction Value: 815
Transaction timestamp: 14
65
Sender Name: C
Receiver Name: W
Transaction Value: 318
Transaction timestamp: 13
66
Sender Name: A
Receiver Name: W
Transaction Value: 309
Transaction timestamp: 12
67
Sender Name: C
Receiver Name: Z
Transaction Value: 697
Transaction timestamp: 11
68
Sender Name: A
Receiver Name: W
Transaction Value: 285
Transaction timestamp: 10
69
Sender Name: A
Receiver Name: X
Transaction Value: 882
Transaction timestamp: 9
70
Sender Name: B
Receiver Name: Z
Transaction Value: 669
Transaction timestamp: 8
71
Sender Name: C
Receiver Name: W
Transaction Value: 509
Transaction timestamp: 7
72
Sender Name: B
Receiver Name: X
Transaction Value: 166
Transaction timestamp: 6
73
Sender Name: C
Receiver Name: Y
Transaction Value: 500
Transaction timestamp: 5
74
Sender Name: B
Receiver Name: Z
Transaction Value: 610
Transaction timestamp: 4
75
Sender Name: A
Receiver Name: W
Transaction Value: 994
Transaction timestamp: 3
76
Sender Name: D
Receiver Name: X
Transaction Value: 248
Transaction timestamp: 2
77
Sender Name: C
Receiver Name: W
Transaction Value: 950
Transaction timestamp: 1